### DISCUSSION TITLE
How to add histogram plot?

### DISCUSSION THREAD

**Discussion by Murali-001** [Budibase/budibase#9276] [Help] [answered]
I need histogram plot for my data. How do I achieve that with budibase app?

**Comment by melohagan** [comment, marked as answer]
Hey @Murali-001 

Currently this is not supported natively in Budibase, but I have raised an issue here: https://github.com/Budibase/budibase/issues/9294

If you need it ASAP, then you could build a custom component: https://docs.budibase.com/docs/custom-component
That would also allow you to use any svelte library you wanted for your histogram.
I even found this blog where someone built a histogram chart from scratch: https://til.bradyjoslin.com/svelte/histogram/

**Reply by Murali-001** [reply]
Thanks for the reply @melohagan .
